public/app/sw-register.js

```
@@ -1,107 +1,89 @@
-
-    // If a SW is already controlling, we're done immediately.
-    if (navigator.serviceWorker.controller) return reg;
-
-    // Wait for the new SW to take control so model fetches are intercepted.
-    await new Promise((resolve) => {
-      const ctrl = navigator.serviceWorker;
-      if (ctrl.controller) { resolve(); return; }
-      ctrl.addEventListener('controllerchange', resolve, { once: true });
-    });
+    statusUI?.setStatus(key, 'loading');
+    const response = await fetch(url);
```

ESLint

[error] 'fetch' is not defined.

(no-undef)